### ISSUE TITLE
Running `make cluster-up` cleared kubeconfig

### ISSUE THREAD

**Issue by Spazzy757** [sustainable-computing-io/local-dev-cluster#52]
### What happened?

Firstly, would like to say thank you for this project, it is an incredible idea

When looking into running the the local environment, I ran the `make cluster-up` command. Before running this I had my Kube contexts (found at `~/.kube/config`) setup with various clusters I work with day to day, after running the make command the only context set was the `kind-kind` context. 

### What did you expect to happen?

The `kind-kind` context should be appended to my list of contexts so that I will not have to recreate all my credentials everytime I setup the local environment

### How can we reproduce it (as minimally and precisely as possible)?

You can create a kind cluster:

```bash
kind create cluster test
```

You will see the context:
```bash
$ kubectl config get-contexts
CURRENT   NAME        CLUSTER     AUTHINFO    NAMESPACE
*         kind-test   kind-test   kind-test
```

then run the `make cluster-up` and check the contexts again

```bash
$ kubectl config get-contexts
CURRENT   NAME        CLUSTER     AUTHINFO    NAMESPACE
*         kind-kind   kind-kind   kind-kind
```

### Anything else we need to know?

I know its a nitpick for the development environment and not urgent 😅 but for people unaware who want to try contribute its a little annoying

### Kepler image tag

<details>

</details>

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### Cloud provider or bare metal

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Kepler deployment config

<details>

For on kubernetes:
```console
$ KEPLER_NAMESPACE=kepler

# provide kepler configmap
$ kubectl get configmap kepler-cfm -n ${KEPLER_NAMESPACE} 
# paste output here

# provide kepler deployment description
$ kubectl describe deployment kepler-exporter -n ${KEPLER_NAMESPACE} 
```

For standalone:
# put your Kepler command argument here

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by rootfs** [comment]
transferring the issue here

**Comment by marceloamaral** [comment]
@Spazzy757 sorry for the inconvenience.
I create the PR #53 to append the new Kepler kind cluster the the kubeconfig file.
The new cluster will be called `kind-kepler`.

@SamYuan1990 the update should not affect the CI...

**Comment by Spazzy757** [comment]
Thank you so much, really quick turn around.

**Comment by rootfs** [comment]
closed by #53
